docs/API.md

```diff
@@ -979,26 +979,26 @@ Example response:
 
 An object with the following properties:
 
-* `"username"`: An *username*. The name of the account of an authorized user.
-* `"api_key"`: An *api key*. The api_key associated with the username.
-* `"domains"`: A list of *domain names*. The domains to be tested.
-* `"test_params"`: As described below.
+* `"username"`: An *username*, required. The name of the account of an authorized user.
+* `"api_key"`: An *api key*, required. The api_key associated with the username.
+* `"domains"`: A list of *domain names*, required. The domains to be tested.
+* `"test_params"`: As described below, optional. (default with default value as below)
 
 The value of `"test_params"` is an object with the following properties:
 
-* `"client_id"`: A *client id*, optional.
-* `"profile"`: A *profile name*, optional.
-* `"client_version"`: A *client version*, optional.
-* `"nameservers"`: A list of *name server* objects, optional.
-* `"ds_info"`: A list of *DS info* objects, optional.
+* `"client_id"`: A *client id*, optional. (default: unset)
+* `"profile"`: A *profile name*, optional. (default: `"default"`)
+* `"client_version"`: A *client version*, optional. (default: unset)
+* `"nameservers"`: A list of *name server* objects, optional. (default: `[]`)
+* `"ds_info"`: A list of *DS info* objects, optional. (default: `[]`)
 * `"advanced"`: **Deprecated**. A boolean, optional.
-* `"ipv6"`: A boolean, optional. (default: `false`)
-* `"ipv4"`: A boolean, optional. (default: `false`)
-* `"config"`: A string, optional. The name of a *config profile*.
+* `"ipv6"`: A boolean, optional. (default: `true`)
+* `"ipv4"`: A boolean, optional. (default: `true`)
+* `"config"`: A string, optional. The name of a *config profile*. 
```

Comment:
Could you remove the trailing whitespace?

Reply:
done